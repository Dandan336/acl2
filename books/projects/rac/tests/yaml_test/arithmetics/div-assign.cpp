// RAC begin

int div(int a)
{
  a /= 2;
  return a;
}

// RAC end
