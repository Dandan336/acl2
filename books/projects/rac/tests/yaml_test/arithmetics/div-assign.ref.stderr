div-assign.cpp:5: syntax error, unexpected '='
